docs/guide/humans.md

```diff
@@ -0,0 +1,281 @@
+The example worlds can be tested easily; the ".wbt" files are located in various "worlds" directories of the "[WEBOTS\_HOME/projects/humans]({{ url.github_tree }}/projects/humans)" directory and can be directly opened from Webots using the `Open Sample World` item in `File` menu.
+The controller code is located in the corresponding "controllers" directory.
+
+## [skin\_animated\_humans]({{ url.github_tree }}/projects/humans/skin_animated_humans/worlds/skin_animated_humans.wbt) Sample
+
+![skin_animated_humans.png](images/humans/skin_animated_humans.thumbnail.jpg) This example shows how to use the [Skin](../reference/skin.md) node to animate human models and play [BVH](https://en.wikipedia.org/wiki/Biovision_Hierarchy) animations.
+The [FBX](https://en.wikipedia.org/wiki/FBX) models have been generated using [MakeHuman](http://makehumancommunity.org/) and BVH files are taken from open source databases.
+The [Skin PROTO models]({{ url.github_tree }}/projects/humans/skin_animated_humans/protos/) contain the definition of the appearance declared in the in the FBX files.
+The "[bvh\_animation]({{ url.github_tree }}/projects/humans/skin_animated_humans/controllers/bvh_animation.c)" C controller loads the BVH file, associates it with the [Skin](../reference/skin.md) model and plays the BVH animation by applying the poses to the [Skin](../reference/skin.md) bones.
+
+### CharacterSkin PROTO
+
+Skin node representing a human with predefined mesh and appearance.
+Four different characters are available based on the 'model' field:
+- "Anthony": a boy,
+- "Robert": a men,
+- "Sandra": a woman,
+- "Sophia": a girl.
+
+Derived from [Skin](../reference/skin.md).
+
+```
+CharacterSkin {
+  SFVec3f     translation 0 0 0
+  SFRotation  rotation    0 1 0 0
+  SFVec3f     scale       1 1 1
+  SFString    name        "skin"
+  SFString    model       "Sandra"
+  SFBool      castShadows FALSE
+}
+```
+
+> **File location**: "[WEBOTS\_HOME/projects/humans/skin\_animated\_humans/protos/CharacterSkin.proto]({{ url.github_tree }}/projects/humans/skin_animated_humans/protos/CharacterSkin.proto)"
+
+#### CharacterSkin Field Summary
+
+- `model` : Defines which character should be used.
+This field accepts the following values: "Anthony", "Robert", "Sandra", and "Sophia".
+
+### [bvh\_util]({{ url.github_tree }}/projects/humans/skin_animated_humans/libraries/bvh_util/) Library
+To load and play the BVH file a C utility library is also provided.
+The BVH file defines a skeleton in the form of hierarchy of bones, and a time series of angles for each joint in the skeleton.
+Note that this skeleton is not necessarily the same as the skeleton associated with a [Skin](../reference/skin.md) node, since they come from different sources.
+For this reason, this library also provides functions to adapt and re-target the BVH motion data to the [Skin](../reference/skin.md) model.
+
+---
+
+#### `wbu_bvh_read_file`
+#### `wbu_bvh_cleanup`
```

Comment:
The description of the function are built in another way than in other .md files (like https://cyberbotics.com/doc/reference/accelerometer), with the italic text before the tab-component for example.
I don't know if its matter.

Reply:
Yes, I'm aware of this.
But the documentation of other functions in the reference manual has a complete different structure.
I checked different option and for me this one is the one resulting in a more clear structure.